### DISCUSSION TITLE
What would you like to see in this resources repository?

### DISCUSSION THREAD

**Discussion by FrancescoXX** [FrancescoXX/free-Web3-resources#41] [General]
Let's discuss the possible improvements to this repository.

**Comment by tawseefnabi** [comment]
- we can  add youtube channels which have web3 tutorials
- blogs on web3 projects
-  articles or news for new web3 things

**Reply by FrancescoXX** [reply]
I like these

**Reply by tawseefnabi** [reply]
- job resources
- common interview questions

**Reply by JayrajSingh65** [reply]
i done already

**Comment by FrancescoXX** [comment]
added the youtube section, we can add the other 2 once we have more links
